integration-tests/zendeskRequestAPI.spec.ts

```diff
@@ -78,9 +84,45 @@ describe('Zendesk ticket check', () => {
       'X-Zendesk-Webhook-Signature': signature
     }
   }
+
+  let ticketId: string
+
+  beforeAll(async () => {
+    ticketId = await createZendeskRequest()
+    return ticketId
+  })
+
+  afterAll(async () => {
+    return deleteZendeskTicket(ticketId)
```

Comment:
Don't think we need the return, and this needs to be async
```suggestion
   await deleteZendeskTicket(ticketId)
```